packages/core/saved-objects/core-saved-objects-migration-server-internal/src/actions/clone_index.ts

```diff
@@ -8,7 +8,7 @@
 
 import * as Either from 'fp-ts/lib/Either';
 import * as TaskEither from 'fp-ts/lib/TaskEither';
-import { pipe } from 'fp-ts/lib/pipeable';
+import { pipe } from 'fp-ts/lib/function';
```

Comment:
FYI the pipe will no longer be exposed in `pipeable`.
https://gcanti.github.io/fp-ts/modules/pipeable.ts.html#pipe